modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -5,40 +5,35 @@ This module included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="distr-tracing-jaeger-operator-install_{context}"]
-= Installing the {JaegerName} Operator
+= Installing the {JaegerOperator} Operator
 
-To install {JaegerName}, you use the link:https://operatorhub.io/[OperatorHub] to install the {JaegerName} Operator.
+You can install the {JaegerOperator} Operator through the link:https://operatorhub.io/[*OperatorHub*].
```

Comment:
I suggest removing "You can" and just starting with "Install", This is a procedure for installing it, after all, so if the reader is here I'd think they want to install it.

Reply:
Thank you, I will keep this in mind when writing or updating other installation procedures. This particular product is deprecated, so I was reluctant to use the imperative mood here, and there is also a currently undocumented installation procedure using the CLI.